frontend/src/components/InlineStatusEdit.vue

```diff
@@ -24,9 +25,17 @@ export default class InlineStatusEdit extends Vue {
   @Prop() statusItems!: string[];
   @Prop() keyPath!: string;
 
+  get status() {
+    const { item } = this;
+    return findObjectValue(item, "status");
```

Comment:
I'm not sure that it is proper to just recursively search for a `status` key here. When you instantiate the component you should explicitly specify what is the path to the status.

Reply:
I will do something for this

Reply:
yes, this is proper to recursively search for a status key. It is necessary because the data structure of both SQR and SE are different. I have tried your suggestion but not work. so I made this more generic to use it with any component in future.

Reply:
I got your point but when we use { text: "Status", value: "restoration.status" }, It is not working and we have already `computedComponent`,.so we are not able to do like this.

Reply:
I will use `this.value` in this file as per your suggestion. this is fine and it is really a good suggestion.